recipe/meta.yaml

```
@@ -21,7 +24,7 @@ source:
 
 build:
   number: {{ build_number }}
-  skip: true  # [(linux or win) and cuda_compiler == "None"]
+  skip: True  # [((linux or win) and cuda_compiler == "None") or cuda_compiler_version == "11.2"]
```

Maybe we can keep this as two lines to minimize potential future merge conflicts with the conda-forge feedstock

```suggestion
  skip: True  # [(linux or win) and cuda_compiler == "None"]
  skip: True  # [cuda_compiler_version == "11.2"]
```